iree/compiler/Codegen/LLVMCPU/test/BUILD

```diff
@@ -43,6 +44,7 @@ iree_lit_test_suite(
     tools = [
         "//iree/tools:iree-compile",
         "//iree/tools:iree-opt",
+        "//iree/tools:iree-translate",
```

Comment:
We shouldnt have `iree-translate` dependence within `Codegen`. `iree-translate` is for e2e testing. We should be able to do this with `iree-opt`